Content.Server/_Goobstation/Possession/PossessedComponent.cs

```diff
@@ -0,0 +1,68 @@
+// SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+// SPDX-FileCopyrightText: 2025 Solstice <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 SolsticeOfTheWinter <solsticeofthewinter@gmail.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+using Content.Shared.Actions;
+using Content.Shared.Actions.Components;
+using Content.Shared.Mind;
+using Content.Shared.Polymorph;
```

Comment:
Unused using directives
```suggestion
using Content.Shared.Actions.Components;
using Content.Shared.Polymorph;
```